sktime/param_est/seasonality.py

```diff
@@ -407,3 +407,110 @@ def get_test_params(cls, parameter_set="default"):
         params2 = {"candidate_sp": [3, 7, 12]}
 
         return [params1, params2]
+
+
+class SeasonalityPeriodogram(BaseParamFitter):
+    """Score periodicities by their spectral power.
+
+    Interfacing `seasonal.periodogram` to determine candidate seasonality parameters.
+
+    Parameters
+    ----------
+    min_period : int
+        Disregard periods shorter than this number of samples.
+        Defaults to 4
+    max_period : int
+        Disregard periods longer than this number of samples.
+        Defaults to None
+
+    Attributes
+    ----------
+    sp_ : int, seasonality period at lowest p-level, if any sub-threshold, else 1
+        if `candidate_sp` is passed, will be in `candidate_sp` or 1
+    sp_significant_ : list of int, seasonality periods with sub-threshold p-levels
```

Comment:
is this correct, i.e., are they ordered by p-level? If not, the docstring needs to be changed.